packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -66,6 +66,15 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     props.onFetchAvatarPersonaData
   );
 
+  const onRenderAvatar = useCallback(
```

Comment:
@prprabhu-ms for review just to check this doesn't cause the anonymous issue we saw in the previous bug. if it does we do this a lot in out codebase .... 👀